tests/x11/kontact.pm

```diff
@@ -26,14 +26,16 @@ sub run {
     x11_start_program("echo \"[General]\" >> ~/.config/kmail2rc",                    valid => 0);
     x11_start_program("echo \"first-start=false\" >> ~/.config/kmail2rc",            valid => 0);
 
-    my @tags = qw(test-kontact-1 kontact-import-data-dialog kontact-window);
+    my @tags = qw(test-kontact-1 kontact-import-data-dialog kontact-window kontact-error);
     x11_start_program('kontact', target_match => \@tags);
     do {
         assert_screen \@tags;
         # kontact might ask to import data from another mailer, don't
         wait_screen_change { send_key 'alt-n' } if match_has_tag('kontact-import-data-dialog');
         # KF5-based account assistant ignores alt-f4
         wait_screen_change { send_key 'alt-c' } if match_has_tag('test-kontact-1');
+        # workaround for fatal error bsc#1105207
+        wait_screen_change { record_soft_failure('encounter fatal error, bsc#1105207') && return } if match_has_tag('kontact-error');
```

Comment:
Hmm, I wonder why do we need wait_screen_change here if we do early return?

Reply:
I think the problem is that we don't know when fatal error encounters, so if this happens it means screen got updated and then check this change:

http://e13.suse.de/tests/10470#step/kontact/15
(without workaround)

Reply:
thanks, PR updated now